pom.xml

```diff
@@ -198,7 +198,6 @@
                     <redirectTestOutputToFile>true</redirectTestOutputToFile>
                     <runOrder>failedfirst</runOrder>
                     <argLine>
-                        -Xms128m -Xmx1G
```

Comment:
We limit the memory also in the main repo. I think there are 2 reasons for this - 1 in main repo we run multiple surefire forks, so limiting memory ensures less total memory is used. This is not applicable here.

The other reason why I think it is useful - sometimes a new feature or test may leak some garbage and with a memory limit we might discover it sooner. 

We may adjust the limit if 1G is too low.

Reply:
The limit is fine, my reasoning was to keep things simple. 1G was fine to run all the tests. Do you think we should re-add it?

Reply:
Yes, let's put it back, feel free to leave out the `-Xms`, that's really not important here.